pkg/testsuite/server_test.go

```diff
@@ -42,6 +43,8 @@ import (
 	"github.com/stretchr/testify/require"
 )
 
+const ForwardingGrantTypePassword = "password"
```

Comment:
only one last small thing, can you move this to constant.go file in this testsuite package, thx!

Reply:
> only one last small thing, can you move this to constant.go file in this testsuite package, thx!

moved